src/transaction.js

```diff
@@ -53,19 +54,18 @@ var RETRY_INFO_KEY = 'google.rpc.retryinfo-bin';
  */
 var DEFAULT_TRANSACTION_TIMEOUT = config.methods.Commit.timeout_millis;
 
-var services = gax.grpc().load(
-  {
-    root: path.resolve(__dirname, '../protos'),
-    file: 'google/rpc/error_details.proto',
-  },
-  'proto',
-  {
-    binaryAsBase64: true,
-    convertFieldsToCamelCase: true,
-  }
+var protoFilesRoot = new gax.grpc.GoogleProtoFilesRoot();
```

Comment:
This is having an issue on CI (https://circleci.com/gh/googleapis/nodejs-spanner/3702): 

```js
1) Transaction
       "before all" hook:
     TypeError: gax.grpc.GoogleProtoFilesRoot is not a function
      at Object.<anonymous> (src/transaction.js:27:141)
      at Module.replacementCompile (node_modules/nyc/node_modules/append-transform/index.js:58:13)
      at module.exports (node_modules/nyc/node_modules/default-require-extensions/js.js:8:9)
      at /home/node/project/node_modules/nyc/node_modules/append-transform/index.js:62:4
      at extensions..js (node_modules/intelli-espower-loader/node_modules/espower-loader/index.js:47:13)
      at Object..js (node_modules/nyc/node_modules/append-transform/index.js:62:4)
      at require.extensions.(anonymous function) (node_modules/proxyquire/lib/proxyquire.js:305:43)
      at Object.<anonymous> (node_modules/nyc/node_modules/append-transform/index.js:62:4)
      at Proxyquire._withoutCache (node_modules/proxyquire/lib/proxyquire.js:215:12)
      at Proxyquire.load (node_modules/proxyquire/lib/proxyquire.js:129:15)
      at Context.<anonymous> (test/transaction.js:108:19)
```

Reply:
Ok, I changed the way `gax` was being mocked to reflect the new way of loading the proto file.